responsemanager/server.go

```diff
@@ -166,25 +174,36 @@ func (rm *ResponseManager) abortRequest(p peer.ID, requestID graphsync.RequestID
 }
 
 func (rm *ResponseManager) processRequests(p peer.ID, requests []gsmsg.GraphSyncRequest) {
+	ctx, messageSpan := otel.Tracer("graphsync").Start(rm.ctx, "message", trace.WithAttributes(
```

Comment:
rename requestMessage to match corresponding responseMessage on the other side (also cause I'm about to submit a PR that conflicts :)